javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/resolution/MethodResolutionLogic.java

```diff
@@ -613,43 +584,31 @@ private static boolean areOverride(MethodUsage winningCandidate, MethodUsage oth
             return ctx.solveMethod(name, argumentsTypes, staticOnly, typeSolver);
         }
         if (typeDeclaration instanceof JavaParserAnonymousClassDeclaration) {
-        	Context ctx = ((JavaParserAnonymousClassDeclaration) typeDeclaration).getContext();
+            Context ctx = ((JavaParserAnonymousClassDeclaration) typeDeclaration).getContext();
             return ctx.solveMethod(name, argumentsTypes, staticOnly, typeSolver);
         }
         if (typeDeclaration instanceof ReflectionClassDeclaration) {
             return ((ReflectionClassDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
         }
         if (typeDeclaration instanceof ReflectionInterfaceDeclaration) {
-          return ((ReflectionInterfaceDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
+            return ((ReflectionInterfaceDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
         }
-          if (typeDeclaration instanceof ReflectionEnumDeclaration) {
+        if (typeDeclaration instanceof ReflectionEnumDeclaration) {
             return ((ReflectionEnumDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
         }
         if (typeDeclaration instanceof JavassistInterfaceDeclaration) {
             return ((JavassistInterfaceDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
         }
         if (typeDeclaration instanceof JavassistClassDeclaration) {
-          return ((JavassistClassDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
+            return ((JavassistClassDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
         }
-          if (typeDeclaration instanceof JavassistEnumDeclaration) {
+        if (typeDeclaration instanceof JavassistEnumDeclaration) {
             return ((JavassistEnumDeclaration) typeDeclaration).solveMethod(name, argumentsTypes, staticOnly);
         }
         throw new UnsupportedOperationException(typeDeclaration.getClass().getCanonicalName());
     }
 
-    public static SymbolReference<ResolvedMethodDeclaration> solveMethodInFQN(String fqn, String name,
```

Comment:
This is rather internal method, I am not expecting users to invoke this method directly so I think we could remove it.

However this means technically breking the API. What @matozoid think about that? Should we mark it as deprecated instead?

Reply:
If you consider the symbol solver instable: delete it.
If you think nobody is using it: delete it.
Otherwise, deprecate with annotation and javadoc explaining what to use instead.

Reply:
I only introduced this method in #1717, I think we can safely assume that nobody is using it

Reply:
Go ahead :-)